### DISCUSSION TITLE
Image in comment section takes up a lot of space

### DISCUSSION THREAD

**Discussion by mouzedrift** [ppy/osu#20949] [Ideas]
osu web:

![image](https://user-images.githubusercontent.com/43358824/198016436-da0e6447-8ce3-409a-8c0b-192e8d057a01.png)

osu lazer:

https://user-images.githubusercontent.com/43358824/198016648-a231b94e-411f-47db-95f9-d26095876978.mp4

perhaps the image should be scaled down like osu web but that would make the image pretty much unreadable on lazer so maybe there could be some sort of image inspector in lazer (and potentially osu web?) since right clicking on the image and then opening it in another tab doesnt really feel nice

**Comment by bdach** [comment]
Max size should be enforced, but the image should be clickable at most and use the browser to display the image. Adding in-game image preview is overkill IMO.

**Reply by peppy** [reply]
Either that, or clicking should toggle displaying at full size (well, fit to the available width).
